beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestCatalogFixedTermWithVersionedCatalogs.java

```diff
@@ -0,0 +1,132 @@
+package org.killbill.billing.beatrix.integration;
+
+import static org.testng.Assert.assertNotNull;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.joda.time.DateTime;
+import org.joda.time.LocalDate;
+import org.killbill.billing.account.api.Account;
+import org.killbill.billing.api.TestApiListener.NextEvent;
+import org.killbill.billing.beatrix.util.InvoiceChecker.ExpectedInvoiceItemCheck;
+import org.killbill.billing.catalog.api.BillingPeriod;
```

Comment:
This import (`BillingPeriod`) and a few others are not used, could you clean them -- same on your other test `TestCatalogFixedTerm`

Reply:
Fixed